content/09.evolution.md

```diff
@@ -126,31 +126,34 @@ Since then, B.1.1.7 has spread across the UK and internationally, and it has now
+
 The P.1 variant is a sublineage of the B.1.1.28 lineage that was first detected in Japan in samples obtained from four travelers from Brazil during a screening at a Tokyo airport on January 10, 2021 [@url:https://www.ecdc.europa.eu/sites/default/files/documents/COVID-19-risk-related-to-spread-of-new-SARS-CoV-2-variants-EU-EEA-first-update.pdf].
-Shortly thereafter, it was established that there was a concentration of cases of the P.1 variant in Manaus, in the Amazonas State, Brazil.
+Shortly thereafter, it was established that there was a concentration of cases of the P.1 variant in Manaus, Brazil.
 In a small number of samples (n=31) sequenced in Manaus, 42% were identified as the P.1 variant as early as mid-December, but the variant seemed to be absent in genome surveillance testing prior to December [@url:https://virological.org/t/genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-manaus-preliminary-findings/586].
 To date, at least eight countries have detected the P.1 lineage [@url:https://cov-lineages.org/global_report_P.1.html].
 While the majority of P.1 cases detected internationally have been linked to travel originating from Brazil, the UK has also reported evidence of community transmission detected via routine community sequencing [@url:https://cov-lineages.org/global_report_P.1.html; @url:https://www.reuters.com/article/us-health-coronavirus-britain-variants/uk-detects-77-cases-of-south-african-covid-variant-nine-of-brazilian-idUSKBN29T07P].  
 P.1 has eight lineage-specific mutations along with three concerning spike protein mutations in the RBD, including K417T, E484K, and N501Y [@doi:10.1101/2020.12.21.20248640].
  
 There have been multiple different SARS-CoV-2 lineages detected that have mostly been of no more clinical concern than the original devastating lineage originating in Wuhan [@url:https://cov-lineages.org/lineages.html].
 However, the spotlight has been cast on other variants of unknown clinical relevance due to the increase of cases observed that have been associated with B.1.1.7 in particular.  
-Although early in its ascendency, B.1.427/429 is SARS-CoV-2 variants that was detected in California, USA and also known as CAL.20C [@doi:10.1101/2021.01.18.21249786].
+Although early in its ascendency, B.1.427/429 are SARS-CoV-2 variants that was detected in California, USA and also known as CAL.20C [@doi:10.1101/2021.01.18.21249786].
 It was first detected in July 2020 but was not detected again until October 2020. 
 In December 2020, B.1.427/429 accounted for ~24% of the total cases in Southern California and ~36% of total cases in the Los Angeles area. 
-B.1.427/429 has now been detected in New York and Washington, D.C., and some cases have been reported outside the USA in Oceania [@doi:10.1101/2021.01.18.21249786].
+B.1.427/429 have now been detected in New York and Washington, D.C., and some cases have been reported outside the USA in Oceania [@doi:10.1101/2021.01.18.21249786].
```

Comment:
```suggestion
B.1.427/429 have now been detected in several U.S. states and at least 38 countries worldwide [@url:https://www.gisaid.org/hcov19-variants/; @doi:10.1101/2021.01.18.21249786].
```

Reply:
@rando I think you had mentioned a phylogenetic tree potentially, there seems to be some available with Gisaid
https://www.gisaid.org/hcov19-variants/